src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -1098,6 +1098,7 @@ public sealed partial class Utils
     {
         internal Utils() { }
         public static System.Array CopyArray(System.Array arySrc, System.Array aryDest) { throw null; }
+        public static string GetResourceString(string ResourceKey, params string[] Args) { throw null; }
```

Comment:
@cston since this API is being exposed publicly, please make sure its definition is documented with triple slash comments (or the equivalent for vb) so we can automatically port that documentation to the dotnet-api-docs repo when the assembly gets officially updated.